DrDocket/Date.cpp

```diff
@@ -46,13 +46,20 @@ Date
 #include <iostream>
 #include "Date.h"
 
-Date::Date(Month m, unsigned int d, unsigned int y) {
+Month Months[13] = {JAN, JAN, FEB, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC};
```

Comment:
Why are there two Jan's?

Reply:
To facilitate looking up the proper enum for months 1, 2, ..., 12, 1 => JAN, 2 => FEB.
With this array you can do properEnum = Months[month]